tools/pepsi/src/power_off.rs

```diff
@@ -11,31 +16,52 @@ use crate::{
 
 #[derive(Args)]
 pub struct Arguments {
-    /// The NAOs to power off e.g. 20w or 10.1.24.22
-    #[arg(long)]
-    pub naos: Option<Vec<NaoAddress>>,
-    /// Power off all NAOs
-    #[arg(long)]
-    pub all: bool,
+    /// The NAOs to power off e.g. 20w, 10.1.24.22 or all
+    #[arg(required = true)]
+    pub naos: Vec<AddressOrAll>,
 }
```

Comment:
suggestion: ```rust
#[derive(Subcommand, Debug)]
pub enum Commands {
    Poweroff {
        #[arg(long)]
        all: bool,
        #[arg(conflicts_with = "all", required = true, num_args = 1..)]
        numbers: Vec<i32>,
    },
}```

Reply:
I really like this approach and have implemented it now!